### ISSUE TITLE
com.lambdaworks.redis.RedisCommandExecutionException: ERR unknown command 'ROLE'

### ISSUE THREAD

**Issue by tdubey** [mp911de/lettuce#405]
Hi,

I am getting exception when I am trying to connect to Redis server. Here is my simple code

```
        List<RedisURI> redisURIList = Arrays.asList(new RedisURI[]{RedisURI.create("172.19.112.249", 6379), RedisURI.create("172.19.112.249", 6380)});
        RedisCommands<String, String> redisCommands = MasterSlave.connect(RedisClient.create(), new Utf8StringCodec(), redisURIList).sync();
```

And following is an exception

```
INFO: An exception was caught and reported. Message: com.lambdaworks.redis.RedisCommandExecutionException: ERR unknown command 'ROLE'
com.lambdaworks.redis.RedisCommandExecutionException: ERR unknown command 'ROLE'
	at com.lambdaworks.redis.LettuceFutures.awaitAll(LettuceFutures.java:67)
	at com.lambdaworks.redis.masterslave.StaticMasterSlaveTopologyProvider.getNodes(StaticMasterSlaveTopologyProvider.java:76)
	at com.lambdaworks.redis.masterslave.MasterSlaveTopologyRefresh.getNodes(MasterSlaveTopologyRefresh.java:64)
	at com.lambdaworks.redis.masterslave.MasterSlave.connectStaticMasterSlave(MasterSlave.java:254)
	at com.lambdaworks.redis.masterslave.MasterSlave.connect(MasterSlave.java:150)
```

I am trying to use Static Master/Slave with predefined node addresses as described in this (https://github.com/mp911de/lettuce/wiki/Master-Slave) document .

What I am doing wrong?

The other thing is, if I provide single RedisURI, it works fine, e.g. following code works just fine
```
RedisCommands<String, String> redisCommands = MasterSlave.connect(RedisClient.create(), new Utf8StringCodec(), RedisURI.create("172.19.112.249", 6379)).sync();

```

**Comment by mp911de** [comment]
Master/Slave with one node uses `INFO REPLICATION` to discover topology while the use of multiple nodes additionally uses the `ROLE` command. This command is available as of Redis  2.8.12. Please check via `redis-cli` whether you can execute `ROLE` on your Redis nodes.

It's possible that `ROLE` is disabled or not available on your Redis server.

**Comment by mp911de** [comment]
Closing this ticket because of inactivity.
